README.md

```diff
@@ -0,0 +1,60 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+![](https://img.shields.io/badge/Academic-blue)
+![](https://img.shields.io/badge/HTML-red)
+![](https://img.shields.io/badge/JavaScript-yellow)
+
+
+>  "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list.
+
+
+## Built With
+
+- Technologies used: HTML, CSS, Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://banzaman.github.io/mv-awesomeBooks.github.io/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+- Click the green `Code` button on the repo
+- In your local PC, open your terminal in the folder you would like to clone the project.
+- Clone the repo with the command: `git clone (copied link)`; like so: `git clone https://github.com/banzaman/mv-awesomeBooks`
+- On the terminal, navigate into the directory like so: `cd mv-awesomeBooks`
+
```

Comment:
- [OPTIONAL] Please it will be nice to complete the steps, up to running the app on the browser as this will be of help to non technical users, more steps like;
- `npm install`,
- `npm start`,

Reply:
thanks for the advice @Mihndim2020